tests/unit_tests/test_CRISPRessoShared.py

```diff
@@ -21,3 +21,34 @@ def test_get_mismatches():
         -3,
     )
     assert len(mismatch_cords) == 6
+
+
+def test_get_relative_coordinates():
+    s1inds_gap_left, s1inds_gap_right = CRISPRessoShared.get_relative_coordinates('ATCGT', 'TTCGT')
+    assert s1inds_gap_left == [0, 1, 2, 3, 4]
+    assert s1inds_gap_right == [0, 1, 2, 3, 4]
+
+
+def test_get_relative_coordinates_to_gap():
+    to_sequence = 'TTC-T'
+    from_sequence = 'TTCGT'
+    s1inds_gap_left, s1inds_gap_right = CRISPRessoShared.get_relative_coordinates(to_sequence, from_sequence)
+    assert s1inds_gap_left == [0, 1, 2, 2, 3] # should this be [0, 1, 2, 2, 4]?
+    assert s1inds_gap_right == [0, 1, 2, 3, 3] # should this be [0, 1, 2, 4, 4]?
+    assert from_sequence[0] == to_sequence[s1inds_gap_left[0]]
+    assert from_sequence[1] == to_sequence[s1inds_gap_left[1]]
+    assert from_sequence[2] == to_sequence[s1inds_gap_left[2]]
+    assert from_sequence[2] == to_sequence[s1inds_gap_left[2]]
+    # assert from_sequence[4] == to_sequence[s1inds_gap_left[4]] # this should accept, right?
```

Comment:
@mbowcut2 do you have an opinion of the questions in the comments here?

Reply:
The rule here is that from_sequence[i] matches to to_sequence[inds[i]]
Yes, should be [0,1,2,2,4] and [0,1,2,4,4]
Yes, `from_sequence[4] == to_sequence[s1inds_gap_left[4]]`

Reply:
I don't understand how that could hold. Like in this test case:

`to_sequence = 'TTC-T'`
`from_sequence = 'TTCGT'`

`s1inds_gap_left = [0, 1, 2, 2, 4]`

`from_sequence[i] == ['T', 'T', 'C', 'G', 'T'] for each i`
`to_sequence[s1inds_gap_left[i]] == ['T', 'T', 'C', 'C', 'T'] for each i``

So they will always differ in the gap. Right?

Reply:
Yes, they will always differ at the gap. But they should point either to the left or right of the gap. Does that make sense?